content/en/docs/migration/ot_overview.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Migrating from OpenTelemetry 
+weight: 20
+---
+
+The OpenTelemetry project aims to provide backwards compatibility with the
+[OpenTracing](https://opentracing.io) project, in order to ease migration to OpenTelemetry. 
+The intention is to allow OpenTracing instrumentation to be
+recorded using the OpenTelemetry SDK, and to allow both the OpenTelemetry 
+and the OpenTracing APIs to be used together in the same codebase.
+
+To accomplish this, each OpenTelemetry SDK provides and **OpenTracing Shim**, which can be installed
+in order to connect the SDK to the OpenTracing API. Please note that the OpenTracing Shim is not
+enabled by default.
+
+## Language version Support
+
+For reference see the following table, listing the OpenTracing
+and OpenTelemetry APIs minimum **language** version, as of
+January 2022:
+
+| Language   | OpenTracing API  | OpenTelemetry API |
+| ---------- | ---------------- | ----------------- |
+| Go         | 1.13             | 1.16              |
+| Java       | 7                | 8                 |
+| Python     | 2.7              | 3.6               |
+| Javascript | 6                | 8.5               |
+| DotNet     | 1.3              | 1.4               |
+| CPP        | 11               | 11                |
+
+Users are also encouraged to check and update their language and runtime
+components before using the Shim layer, as the OpenTelemetry APIs and SDKs
+may have higher version requirements than their OpenTracing counterparts.
+e.g. OpenTracing Python supports Python 2.6 and higher, whereas the
+OpenTelemetry Python API supports Python 3.5 and higher.
+
+## Migration Overview
+
+Currently, there are many codebases which are instrumented with OpenTracing. These
+codebases may use the OpenTracing API to instrument their apllication code, and they may
+install OpenTracing plugins to instrument the libraries and frameworks.
+
+For a simple approach to migrating to OpenTelemetry, you can do the following.
+
+1. Install the OpenTelemetry SDK, and remove the current OpenTracing implementation (e.g. the Jaeger client).
+2. Install the OpenTelemetry instrumentation libraries, and remove the OpenTracing equivalents.
+3. Update your dashboards, alerts, etc, to consume the new OpenTelemetry data.
+4. When writing new application code, write all new instrumentation using the OpenTelemetry API.
+5. Reinstrument your existing application code using the OpenTelemetry API when it is convenient.
+There is no hard requirement to remove existing OpenTracing API calls from your application, they will continue to work.
+
+The above approach works. However, replacing all existing OpenTracing instrumentation libraries at once could be burdensome. 
+To avoid any break in observability, we recommend that OpenTracing users progressively migrate their applications.
+The following migration guide presents a careful, incremental approach to transitioning to OpenTelemetry.
```

Comment:
IMHO, the entire page is a "migration guide". How about replacing the sentence

> The following migration guide presents a careful

with

> The steps below offer an incremental approach to transitioning to OpenTelemetry.